templates/airtable/.nango/schema.ts

```
@@ -0,0 +1,86 @@
+export interface Table {
+  baseId: string;
+  baseName: string;
+  id: string;
+  name: string;
+  views: (string | string | string)[];
+  fields: (string | string | string | string | {})[];
+  primaryFieldId: string;
+};
+
+export interface SyncMetadata_airtable_bases {
+};
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)